tests/unit/modules/test_ansiblegate.py

```diff
@@ -42,9 +43,9 @@ class AnsiblegateTestCase(TestCase, LoaderModuleMockMixin):
     def setUp(self):
         self.resolver = ansible.AnsibleModuleResolver({})
         self.resolver._modules_map = {
-            'one.two.three': '/one/two/three.py',
-            'four.five.six': '/four/five/six.py',
-            'three.six.one': '/three/six/one.py',
+            'one.two.three': os.sep + os.path.join('one', 'two', 'three.py'),
+            'four.five.six': os.sep + os.path.join('four', 'five', 'six.py'),
+            'three.six.one': os.sep + os.path.join('three', 'six', 'one.py'),
```

Comment:
Apology overlooking Windows, since my last use of it was NT 4.0 in 1998 and since that I didn't touched it. So I have indeed no clue how this behaves there. Someone from Windows world should thoroughly look at this, sorry.

I would say, this is completely unfinished fix and means nothing at the moment. The path here is not even used anywhere to look/verify at in these tests. As of now the Ansiblegate _will_ break on Windows, because it is working only with POSIX paths at the moment — they are constructed with `os.listdir` "as is", splitted by [intentionally hard-coded slash](https://github.com/saltstack/salt/blob/oxygen/salt/modules/ansiblegate.py#L85) (so someone will fix this on Windows properly) as just testing internal Python components like `os.listdir` is just pointless. 

So I would rather put "Windows is not supported" in `__virtual__` for now (forgot to do this, even...) and that's it. Ansiblegate is going to have updates soon, but not in the Oxygen.